task9/task9.js

```diff
@@ -0,0 +1,512 @@
+var adList = [
```

Comment:
This script is not even connected to an HTML file, so question is: have you really tested it?